client/me/profile-links/index.jsx

```diff
@@ -123,7 +123,7 @@ class ProfileLinks extends Component {
 				{ times( 2, ( index ) => (
 					<ProfileLink
 						title="Loading Profile Links"
-						url="http://wordpress.com"
+						url="https://wordpress.com/"
```

Comment:
I just wanted to note that this change applies only to logged-out users. For logged-in users, we'll be redirected to our primary site anyway, so a redirect will always be needed.

Reply:
These should be an internal redirect in modern browsers, so it's not quite as bad. But still, there's no reason to keep them around like that 🙂